go/vt/vttablet/tabletserver/tabletserver.go

```diff
@@ -215,7 +215,7 @@ func (tsv *TabletServer) InitDBConfig(target querypb.Target, dbcfgs *dbconfigs.D
 	tsv.txThrottler.InitDBConfig(target)
 	tsv.vstreamer.InitDBConfig(target.Keyspace)
 	tsv.hs.InitDBConfig(target)
-	tsv.lagThrottler.InitDBConfig(target.Keyspace, target.Shard)
+	//tsv.lagThrottler.InitDBConfig(target.Keyspace, target.Shard)
```

Comment:
Please uncomment. Throttler now disabled by default. (merge `master`)

Reply:
Actually I converted this to a draft PR and is superseded by https://github.com/vitessio/vitess/pull/6892. These throttler related changes were not going to be part of the release.